src/main/java/com/atherys/towns/facade/NationFacade.java

```diff
@@ -185,6 +193,84 @@ public void setNationTax(Player source, double tax) throws TownsCommandException
         townsMsg.info(source, "Nation tax changed to: " + tax + ".");
     }
 
+    public void claimNationPlotFromPlayerSelection(Player player, Nation nation) throws CommandException {
+        PlotSelection selection = plotSelectionFacade.getValidPlayerPlotSelection(player);
+
+        NationPlot plot = plotService.createNationPlotFromSelection(selection);
+
+        int plotArea = MathUtils.getArea(plot);
+        if (MathUtils.getArea(plot) < config.TOWN.MAX_PLOT_AREA) {
+            throw new TownsCommandException("Plot selection is smaller than area permitted ( ", plotArea, " < ", config.TOWN.MAX_PLOT_AREA, " )");
+        }
```

Comment:
So this checks against the max plot area of a town to decide if the nation plot is too small?

Reply:
Threw this in because it makes no sense to have a NationPlot smaller than a TownPlot
I could move this to a separate configuration.

Reply:
ah true, ok in that case I say its probably fine as is.